tests/browser_test.go

```diff
@@ -46,6 +50,76 @@ func TestBrowserNewPage(t *testing.T) {
 	assert.Equal(t, 0, l, "expected there to be 0 browser context after second page close, but found %d", l)
 }
 
+func TestBrowserOn(t *testing.T) {
+	t.Parallel()
+
+	script := `
+		b.on('%s').then((val) => {
+			log('ok: '+val);
+		}, (val) => {
+			log('err: '+val);
+		});`
+
+	t.Run("err_wrong_event", func(t *testing.T) {
+		t.Parallel()
+
+		b := newTestBrowser(t)
+		rt := b.vu.Runtime()
+		require.NoError(t, rt.Set("b", b.Browser))
+
+		err := b.vu.loop.Start(func() error {
+			if _, err := rt.RunString(fmt.Sprintf(script, "wrongevent")); err != nil {
+				return fmt.Errorf("%w", err)
+			}
+			return nil
```

Comment:
It was like that originally, but that triggers the `wrapcheck` linter (see [the above comment](https://github.com/grafana/xk6-browser/pull/287#discussion_r838711411)) since `rt.RunString` returns a goja error.